### PR TITLE
Update README.md - Add how to run on K8s

### PR DESCRIPTION
**Pull request by tareksamni**
Update README.md - Add how to run on K8s

This is how I deployed Ciao to my K8s cluster and it worked perfectly 👌So thought it might be good to share it till we publish the Helm chart!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,6 +206,83 @@ Be sure to enable authentication (eg. HTTP Basic auth) and TLS certificates if y
...
+````
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: monitoring
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rails
```
Comment (brotandgames) [inline README.md]:
```suggestion
  name: ciao
```

File: README.md [hunk 2]
```diff
@@ -206,6 +206,83 @@ Be sure to enable authentication (eg. HTTP Basic auth) and TLS certificates if y
...
+        resources:
+          requests:
+            memory: 256Mi
+            cpu: 200m
+          limits:
+            memory: 512Mi
+            cpu: 400m
+        envFrom:
+        - secretRef:
+            name: rails
```
Comment (brotandgames) [inline README.md]:
```suggestion
            name: ciao
```

File: README.md [hunk 3]
```diff
@@ -206,6 +206,83 @@ Be sure to enable authentication (eg. HTTP Basic auth) and TLS certificates if y
...
+kind: Service
+metadata:
+  name: ciao
+  namespace: monitoring
+spec:
+  ports:
+    - port: 80
+      targetPort: 3000
+      protocol: TCP
+  type: NodePort
```
Comment (brotandgames) [inline README.md]:
```suggestion
  type: ClusterIP
```

**Comment by tareksamni** [comment]
@brotandgames all good now?

**Comment by tareksamni** [comment]
@brotandgames sorry for the force-push 😱I recommitted your changes again.

**Comment by brotandgames** [comment]
@tareksamni no problem. Thanks!

**Pull request merged**